actioncable/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Bring `ActionCable::Connection::TestCookieJar` in alignment with `ActionDispatch::Cookies::CookieJar` in regards to setting the cookie value.
```

Comment:
```suggestion
## Unreleased

*   Bring `ActionCable::Connection::TestCookieJar` in alignment with `ActionDispatch::Cookies::CookieJar` in regards to setting the cookie value.
```

Might need to add a heading here, waiting on CI (or you can run `bundle exec tools/railspect changelogs .` locally to verify)

Reply:
:green_circle: It passed: https://buildkite.com/rails/rails/builds/107905#018fd0ea-230e-4ea1-808c-3276e7798238

Reply:
We don't use that heading. The header is added on the release